tests/testthat/test-plot_alignment_table.R

```diff
@@ -4,9 +4,10 @@ test_that("if `data` is not a data frame errors gracefully", {
   expect_snapshot_error(plot_alignment_table(1))
 })
 
-test_that("if `data` has zero rows errors gracefully", {
+test_that("if `data` has zero rows returns empty plot", {
   zero_row <- toy_data_alignment_table[0L, ]
-  expect_snapshot_error(plot_alignment_table(zero_row))
+  p <- plot_alignment_table(zero_row)
+  expect_true(length(p$data) == 0)
```

Comment:
NIT: this expectation could be more precise with something like (similar comment for other equivalent tests): 
``` r
expect_equal(p, empty_plot_no_data_message() )
```